NorthstarDLL/mods/modmanager.cpp

```diff
@@ -58,8 +58,6 @@ Mod::Mod(fs::path modDir, char* jsonBuf)
 
 	Name = modJson["Name"].GetString();
 
-	spdlog::info("Loading mod '{}'", Name);
-
```

Comment:
Is there a reason why this bit was removed? 

originally I added it so that the logs are more obvious to people reading them, because people tend to interpret
```
...
loaded convar 1
loaded convar 2
loaded mod 1
loaded convar 3
loaded convar 4
loaded mod 2
...
```
as mod 1 owning convar 3 and 4, compared to
```
...
loading mod 1
loaded convar 1
loaded convar 2
loaded mod 1
loading mod 2
loaded convar 3
loaded convar 4
loaded mod 2
...
```

this isn't really a problem for the most part, because I also log the owning mod for things like convar registration, however for malformed mods which produce warnings, the owning mod is not logged. I would be fine with either keeping this bit, or logging the owning mod in the warnings as well

Reply:
Forgot to re-add when reverting the disabled mod skipping logic